ambari-server/src/main/python/ambari_server/setupSecurity.py

```diff
@@ -637,20 +684,16 @@ def setup_ldap(options):
 
   ldap_property_list_reqd = init_ldap_properties_list_reqd(properties, options)
 
-  ldap_property_list_opt = ["authentication.ldap.managerDn",
+  ldap_property_list_opt = ["ambari.ldap.connectivity.bind_dn",
```

Comment:
There should be a constant created for "ambari.ldap.connectivity.bind_dn" to help avoid typo errors.